extension/src/test/fixtures/expShow/base/customPlots.ts

```diff
@@ -30,6 +30,30 @@ export const experimentsWithCommits: Experiment[] = [
     },
     params: { 'params.yaml': { dropout: 0.122, epochs: 5 } }
   },
+  {
+    branch: 'main',
+    id: 'fe2919b',
```

Comment:
[F] There are changes to tests and fixtures because I added two more commits to the base fixture.